src/ansys/fluent/interface/post_objects/post_objects_container.py

```diff
@@ -0,0 +1,248 @@
+        the container to the parent object (`obj`). A `create()` method is also
+        dynamically added to each container for creating and initializing new objects.
+        """
+        # Iterate through all attributes in the module's dictionary
+        for name, cls in mod.__dict__.items():
+            if cls.__class__.__name__ in (
+                "PyLocalNamedObjectMetaAbstract",
+            ) and not inspect.isabstract(cls):
+                cont = PyLocalContainer(self, cls, post_api_helper, cls.PLURAL)
+
+                # Define a method to add a "create" function to the container
+                def _add_create(py_cont):
+                    def _create(**kwargs):
+                        new_object = py_cont.__getitem__(
+                            py_cont._get_unique_chid_name()
+                        )
+                        # Validate that all kwargs are valid attributes for the object
+                        unexpected_args = set(kwargs) - set(new_object())
+                        if unexpected_args:
+                            raise TypeError(
+                                f"create() got an unexpected keyword argument '{next(iter(unexpected_args))}'."  # noqa: E501
+                            )
+                        for key, value in kwargs.items():
+                            setattr(new_object, key, value)
+                        return new_object
+
+                    return _create
+
+                # Attach the create method to the container
+                setattr(cont, "create", _add_create(cont))
+                # Attach the container to the parent object
+                setattr(
+                    obj,
+                    cls.PLURAL,
+                    cont,
+                )
+
+
+class Plots(Container):
+    """Provides the Matplotlib ``Plots`` objects manager.
+
+    This class provides access to ``Plots`` object containers for a given
+    session so that plots can be created.
+
+    Parameters
+        ----------
+        session : obj
+            Session object.
+        module: object
+            Python module containing post definitions
+        post_api_helper: object
+            Provides helper APIs for post-processing
+        local_surfaces_provider : object, optional
+            Object providing local surfaces so that you can access surfaces
+            created in other modules, such as pyvista. The default is ``None``.
+
+    Attributes
+    ----------
+    XYPlots : dict
+        Container for XY plot objects.
+    MonitorPlots : dict
+        Container for monitor plot objects.
+    """
+
+    _sessions_state = {}
+
+    def __init__(self, session, module, post_api_helper, local_surfaces_provider=None):
+        """__init__ method of Plots class."""
+        super().__init__(
+            session, self.__class__, module, post_api_helper, local_surfaces_provider
+        )
+
+
+class Graphics(Container):
+    """Provides the pyvista ``Graphics`` objects manager.
```

Comment:
@prmukherj is the reference to pyvista appropriate here?

Reply:
Nope, you are right. Will update it.

Reply:
Done.